pkg/logs/config/config.go

```diff
@@ -281,3 +285,26 @@ func TaggerWarmupDuration() time.Duration {
 func AggregationTimeout() time.Duration {
 	return defaultLogsConfigKeys().aggregationTimeout()
 }
+
+func getBackupEndpoint(logsConfig *LogsConfigKeys, main Endpoint) *Endpoint {
+	var backup *Endpoint
+	backupEndpoint := main
+
+	if logsDDBackupURL, logsDDURLDefined := logsConfig.logsDDURLBackup(); logsDDURLDefined {
```

Comment:
```suggestion
	if logsDDBackupURL, defined := logsConfig.logsDDURLBackup(); defined {
```
this also seems inconsistent between "dd-backup-url" and "dd-url-backup"